packages/nimble-components/src/table/tests/table-grouping.spec.ts

```diff
@@ -649,4 +655,134 @@ describe('Table grouping', () => {
             ]);
         });
     });
+
+    describe('collapse all button:', () => {
```

Comment:
If it isn't already covered by an existing test, can you add a test that verifies that a data update via `setData()` keeps existing groups collapsed but has new groups expanded?

Reply:
I think [this test](https://github.com/ni/nimble/blob/228177017cae5427fd0bb3a535b0087c17c3bb32/packages/nimble-components/src/table/tests/table-grouping.spec.ts#L135) does what you ask.

Reply:
Could we add a similar test to the one you referenced that uses the "Collapse All" button? That would verify that the updates to `collapsedRows` are made correctly when the "Collapse All" button is pressed.